src/app/campaign/directives/campaign-script.component.css

```diff
@@ -0,0 +1,19 @@
+.sponsor-provided {
+  font-weight: 400;
```

Comment:
This is a separate concern, but we should start forming better habits about identifying things that should be handled by a styleguide. Font weights and sizes in component css is one of them.

Reply:
👍